### PR TITLE
Feature/cosmetics

### PR DESCRIPTION
**Pull request by garyleung142857**
Have been working on the looks of WalletStatus, VoteList, and New Vote.

Will work on updating the cosmetics of Admin page and Vote Page.

### CODE REVIEW

File: components/WalletStatus.vue [hunk 1]
```diff
@@ -31,19 +54,32 @@ export default {
...
+                // eslint-disable-next-line no-console
+                console.error(error);
+                this.error = error;
+            }
         },
         async onClickConnectWallet() {
             let res = false;
             try {
                 res = await this.initWallet();
+                this.error = ''
```
Comment (williamchong) [inline components/WalletStatus.vue:82]: we should reset `this.error` before `await`?
```suggestion
                this.error = ''
                res = await this.initWallet();
```

File: components/WalletStatus.vue [hunk 2]
```diff
@@ -31,19 +54,32 @@ export default {
...
         this.showInstallMetaMask = !window.ethereum;
     },
     methods: {
         ...mapActions('wallet', ['initWallet', 'switchNetwork']),
         async onChangeNetwork(e) {
             this.networkValue = e;
-            await this.switchNetwork(e);
+            try {
+                await this.switchNetwork(e);
+                this.error = ''
```
Comment (williamchong) [inline components/WalletStatus.vue:71]: we should reset `this.error` before `await`?
```suggestion
                this.error = ''
                await this.switchNetwork(e);
```

File: layouts/default.vue [hunk 3]
```diff
@@ -66,11 +63,11 @@ export default {
...
-                {
-                    icon: "mdi-chart-bubble",
-                    title: "Demo Vote",
-                    to: `/vote/${contractAddress.eVote}`,
-                },
+                // {
+                //     icon: "mdi-chart-bubble",
+                //     title: "Demo Vote",
+                //     to: `/vote/${contractAddress.eVote}`,
+                // },
```
Comment (williamchong) [inline layouts/default.vue:70]: We can just remove it

File: pages/setup/index.vue [hunk 4]
```diff
@@ -1,25 +1,59 @@
...
+                />
             </div>
+            <h2>
+                Intervals
+                <v-tooltip right>
+                    <template #activator="{ on }">
+                        <v-icon small v-on="on">mdi-information-outline</v-icon>
+                    </template>
+                    <div>1 Ethereum Block = 12 seconds</div>
+                    <div>1 Polygon Block = 5 seconds</div>
```
Comment (williamchong) [inline pages/setup/index.vue:20]: We can get current network from store, then get the block time info from the network objects as in `constant`

File: pages/setup/index.vue [hunk 5]
```diff
@@ -108,15 +159,27 @@ export default {
...
             } catch (err) {
+                // eslint-disable-next-line no-console
                 console.error(err);
                 this.error = err;
             } finally {
                 this.isLoading = false;
             }
         },
+        voterListRule(list){
+            if(list.length===0) return 'There must be at least one voter.'
```
Comment (williamchong) [inline pages/setup/index.vue:174]:
```suggestion
            if (list.length===0) return 'There must be at least one voter.'
```
Not sure if linter is working correctly, can you try running `lint` in npm and possibly `--fix`?

**Review by williamchong — CHANGES_REQUESTED** [review]
We can deploy any ui change once its ready, no need to wait for all pages to be ready for merging

File: pages/setup/index.vue (hunk 5 above)
Comment (garyleung142857) [reply pages/setup/index.vue:174]: Eslint did not point out this. I will just add the space for now.

File: pages/setup/index.vue (hunk 4 above)
Comment (garyleung142857) [reply pages/setup/index.vue]: Done, please check [commit](https://github.com/DASC7600-2022-E-Voting/E-Voting-App/pull/23/commits/f7bfe5bd9ebf5251348becabff412214d8d0a0ad).

**Review by williamchong — APPROVED** [review]

**Pull request merged**